src/firefly/js/ui/ImageSearchPanelV2.jsx

```diff
@@ -0,0 +1,432 @@
+                                onCancel = {hideSearchPanel}>
+                        <ImageType/>
+                        { isThreeColor && <ThreeColor {...{imageMasterData, multiSelect}}/>}
+                        {!isThreeColor && <SingleChannel {...{groupKey: FG_KEYS.single, imageMasterData, multiSelect}}/>}
+                    </FormPanel>
+                </div>
+            );
+        } else {
+            return <div className='ImageSearch__mask'> <div className='loading-mask'/> </div>;
+        }
+    }
+}
+
+ImageSearchPanelV2.propTypes = {
+    title: PropTypes.string,
+    multiSelect: PropTypes.bool
+};
+
+
+function SingleChannel({groupKey, imageMasterData, multiSelect}) {
+    return (
+        <div style={{width:'100%'}}>
+            <FieldGroup groupKey={groupKey} reducerFunc={mainReducer} keepState={true}>
+                <ImageSource {...{groupKey, imageMasterData, multiSelect}}/>
+            </FieldGroup>
+        </div>
+    );
+}
+
+function ThreeColor({imageMasterData, multiSelect}) {
+
+    return (
+        <div style={{marginTop: 5}}>
+            <Tabs componentKey='ImageSearchPanelV2' resizable={false} useFlex={true}
+                  borderless={true} contentStyle={{backgroundColor: 'rgb(202, 202, 202)', paddingBottom: 2}}
+                  headerStyle={{display:'inline-flex', justifyContent:'center'}}>
+                <Tab key='ImageSearchRed' name='red' label={<div style={{width:40, color:'red'}}>Red</div>}>
+                    <SingleChannel {...{groupKey: FG_KEYS.red, imageMasterData, multiSelect}}/>
+                </Tab>
+                <Tab key='ImageSearchRed' name='green' label={<div style={{width:40, color:'green'}}>Green</div>}>
+                    <SingleChannel {...{groupKey: FG_KEYS.green, imageMasterData, multiSelect}}/>
+                </Tab>
+                <Tab key='ImageSearchRed' name='blue' label={<div style={{width:40, color:'blue'}}>Blue</div>}>
+                    <SingleChannel {...{groupKey: FG_KEYS.blue, imageMasterData, multiSelect}}/>
+                </Tab>
+            </Tabs>
+        </div>
+    );
+}
+
+function ImageType({}) {
+    return (
+        <FieldGroup className='ImageSearch__section' groupKey={FG_KEYS.image_type} keepState={true}>
+            <div className='ImageSearch__section--title'>1. Choose image type</div>
+            <RadioGroupInputField
+                initialState= {{ defaultValue: 'singleChannel',
+                             tooltip: 'Please select the image type'}}
+                options={[  {label: 'Single channel image', value: 'singleChannel'},
+                        {label: '3-color', value: 'threeColor'}]}
+                fieldKey='imageType'
+            />
+        </FieldGroup>
+    );
+}
+
+function ImageSource({groupKey, imageMasterData, multiSelect}) {
+    const isThreeColor = getFieldVal(FG_KEYS.image_type, 'imageType') === 'threeColor';
+    const options = [   {label: 'IRSA', value: 'irsa'},
```

Comment:
ISRA needs to be set-able from a property,  I would prefer it say `Archive` and the property sets it to `IRSA`